src/components/listGames.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import List from './list';
+import ItemGames from './itemGames';
+
+const ListGames = ({ games, onItemSelected }) => {
+    const handleClick = value => {
+        onItemSelected(value);
+    };
+
+    return (
+        <List items={games} title="Open multiplayer games">
+            <ItemGames onClick={handleClick} />
```

Comment:
You shoud use "onItemSelected" directly here instead of create a new function at each render.